Source/Core/Core/DolphinAnalytics.cpp

```diff
@@ -133,20 +133,27 @@ void DolphinAnalytics::ReportGameStart()
 }
 
 // Keep in sync with enum class GameQuirk definition.
-constexpr std::array<const char*, 14> GAME_QUIRKS_NAMES{"icache-matters",
-                                                        "directly-reads-wiimote-input",
-                                                        "uses-DVDLowStopLaser",
-                                                        "uses-DVDLowOffset",
-                                                        "uses-DVDLowReadDiskBca",
-                                                        "uses-DVDLowRequestDiscStatus",
-                                                        "uses-DVDLowRequestRetryNumber",
-                                                        "uses-DVDLowSerMeasControl",
-                                                        "uses-different-partition-command",
-                                                        "uses-di-interrupt-command",
-                                                        "mismatched-gpu-texgens-between-xf-and-bp",
-                                                        "mismatched-gpu-colors-between-xf-and-bp",
-                                                        "uses-uncommon-wd-mode",
-                                                        "uses-wd-unimplemented-ioctl"};
+constexpr std::array<const char*, 19> GAME_QUIRKS_NAMES{
+    "icache-matters",
+    "directly-reads-wiimote-input",
+    "uses-DVDLowStopLaser",
+    "uses-DVDLowOffset",
+    "uses-DVDLowReadDiskBca",
+    "uses-DVDLowRequestDiscStatus",
+    "uses-DVDLowRequestRetryNumber",
+    "uses-DVDLowSerMeasControl",
+    "uses-different-partition-command",
+    "uses-di-interrupt-command",
+    "mismatched-gpu-texgens-between-xf-and-bp",
+    "mismatched-gpu-colors-between-xf-and-bp",
+    "uses-uncommon-wd-mode",
+    "uses-wd-unimplemented-ioctl",
+    "uses-unknown-bp-command",
```

Comment:
Probably not, at least for these strings, since the old ones can't be changed.  The programmer-facing enum could be adjusted, and there could be friendly names and categorizations on a site that actually displays the analytics though. (I don't think there's any site that actually displays them in a friendly way currently.)

Reply:
Yeah I was thinking of doing some server-side postprocessing but I guess it's saner to just leave the old strings untouched...